scripts/build-mv-service

```diff
@@ -49,13 +53,22 @@ def build_mv_service(
     prod_environment = 'prod'
     stage_environment = 'stage'
 
+    tags = list()
```

Comment:
An empty list should be `[]`, but you could probably just make it

```
tags = [{'name':'chef:hudl:mongos_preference', 'value':mongos_preference}]
```

and remove the next line.